google-cloud-clients/pom.xml

```diff
@@ -414,6 +414,7 @@
   <modules>
     <module>google-cloud-automl</module>
     <module>google-cloud-bigtable</module>
+    <module>google-cloud-bigtable-admin</module>
```

Comment:
You also need to update the Stub list further down

Reply:
Not sure if I did it correctly. The list already included
`com.google.cloud.bigtable.admin.v2.stub:com.google.cloud.bigtable.data.v2.stub`

but `com.google.cloud.bigtable.data.v2.stub` had a `*`, which I removed

Reply:
The `*` actually needs to stay, because of the subpackages `mutaterows` and `readrows`

Reply:
fixed